apps/election-manager/src/lib/votecounting.ts

```diff
@@ -325,8 +304,8 @@ export function tallyVotesByContest({
     ) {
       const tallies: Dictionary<ContestOptionTally> = {}
       if (contest.type === 'yesno') {
-        ;[['yes'] as YesNoOption, ['no'] as YesNoOption].forEach(
-          (option: YesNoOption) => {
+        ;[['yes'] as YesNoVoteOption, ['no'] as YesNoVoteOption].forEach(
+          (option: YesNoVoteOption) => {
```

Comment:
This could be done less verbosely like so:

```ts
;([['yes'], ['no']] as YesNoVoteOption[]).forEach((option) => {
  if (option.length === 1) {
    tallies[option[0]] = { option, tally: 0 }
  }
})
```

Or, even more:

```ts
;([['yes'], ['no']] as const).forEach((option) => {
  tallies[option[0]] = { option, tally: 0 }
})
```

That one requires that `YesNoVoteOption` be defined like so, which we probably should do anyway:

```ts
export type YesNoVoteOption = readonly ['yes'] | readonly ['no'] | readonly []
```

Reply:
Since I was just renaming a variable here I'm going to leave as is for now.